queue_services/entity-emailer/src/entity_emailer/email_processors/__init__.py

```diff
@@ -183,11 +185,14 @@ def substitute_template_parts(template_code: str) -> str:
         'initiative-notice',
         'logo',
         'pdf-notice',
-        'style',
         'divider',
+        '8px',
+        '16px',
         '20px',
+        '24px',
         'whitespace-16px',
-        'whitespace-24px'
+        'whitespace-24px',
+        'style'
```

Comment:
There's a difference between, say, `16px` and `whitespace-16px`. If you want me to resolve this (which touches a lot of files), let me know.

I moved `style` last because it has no dependencies.